src/utils/files.ts

```diff
@@ -71,3 +75,10 @@ export const updateBlockSettingsFile = (latestData: UpdateData): void => {
     const updatedDataString = createDataString(latestData);
     writeFileUtil(BLOCK_SETTINGS_FILE, updatedDataString);
 };
+
+export const isLoggedInOrExit = (): void => {
+    if (!existsSync(RC_FILE_PATH)) {
+        logError("You have been logged out. Please run element login");
```

Comment:
Add single quotes around `element login` to help clarify that's the actual command.

```suggestion
        logError("You have been logged out. Please run 'element login'");
```